pubtools/_OIIInspector/_OIIInspector_calls.py

```diff
@@ -0,0 +1,57 @@
+from utils.py import setup_arg_parser
+
+
+
+GET_BUNDLE_IMAGE_OPERATORS_LIST_ARGS[("--name",)]={
+    "help": "run with --name [PARAMETER_1] where PARAMETER_1 is name of the bundle image",
+    "required":True,
+    "type": str,
+}
+
+GET_INDEX_IMAGE_OPERATORS_LIST_ARGS[("--url",)]={
```

Comment:
I'm not sure what these entrypoints are, but I would start with something simple, for example wrapping "ListPackages" grpcurl call. It would take one parameter, the index image address.